registry/deploying.md

```diff
@@ -214,15 +394,68 @@ registry:
     - /path/auth:/auth
 ```
 
-> **Warning**:
-> replace `/path` by whatever directory that holds your `certs` and `auth` folder from above.
+Replace `/path` with the directory which contains the `certs/` and `auth/`
+directories.
 {:.warning}
 
-You can then start your registry with a simple
+Start your registry by issuing the following command in the directory containing
+the `docker-compose.yml` file:
+
+```bash
+$ docker-compose up -d
+```
+
+## Considerations for air-gapped registries
+
+You can run a registry in an environment with no internet connectivity.
+However, if you rely on any images which are not local, you need to consider the
+following:
+
+- You may need to build your local registry's data volume on a connected
+  host where you can run `docker pull` to get any images which are available
+  remotely, and then migrate the registry's data volume to the air-gapped
+  network.
+
+- Certain images, such as the official Microsoft Windows base images, are not
+  distributable. This means that when you push an image based on one of these
+  images to your private registry, the non-distributable layers are **not**
+  pushed, but are always fetched from their authorized location. This is fine
+  for internet-connected hosts, but will not work in an air-gapped set-up.
+
+  In Docker 17.06 and higher, you can configure the Docker daemon to allow
+  pushing non-distributable layers to private registries, in this scenario.
+  **This is only useful in air-gapped set-ups in the presence of
```

Comment:
_Strictly_ speaking, this is also useful in bandwidth-constrained setups, or setups where one wants to not be dependent on fetching the base layers from MS servers. I have no quarrel with the current formulation though.